### PR TITLE
Dev1 - handle tabMenu

### PR DESCRIPTION
**Pull request by congtran8991**
Handle TabMenu

### CODE REVIEW

File: ecommerge_client/basicComponents/BoxWrapper.tsx [hunk 1]
```diff
@@ -8,19 +8,27 @@ const theme = {
   },
 };
 
-interface props {
+interface propsType {
```
Comment (nguyenhoaidanh) [inline ecommerge_client/basicComponents/BoxWrapper.tsx:11]: type vs interface => PascalCase 
`propsType => PropsType`

File: ecommerge_client/basicComponents/BoxWrapper.tsx [hunk 2]
```diff
@@ -8,19 +8,27 @@ const theme = {
...
   children: JSX.Element;
+  width: any;
```
Comment (nguyenhoaidanh) [inline ecommerge_client/basicComponents/BoxWrapper.tsx:13]: don't use any

File: ecommerge_client/components/TabMenu.tsx [hunk 3]
```diff
@@ -1,15 +1,29 @@
+import React, { HtmlHTMLAttributes, useState } from "react";
+import { useRouter } from 'next/router'
 import { Box, List, ListItem } from "@chakra-ui/react";
 import MenuItem from "./item/MenuItem";
-export default function TabMenu() {
+import { tabMenu } from "../constants";
+
+export default function TabMenu(props: any) {
```
Comment (nguyenhoaidanh) [inline ecommerge_client/components/TabMenu.tsx:7]: don't use `any`

File: ecommerge_client/constants/index.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import tabMenu from "constants/tabMenu";
+
+export {
+    tabMenu
```
Comment (nguyenhoaidanh) [inline ecommerge_client/constants/index.ts:4]: MENU_ITEMS

File: ecommerge_client/pages/customer/[pid].tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  const [keyActiveTab, setKeyActiveTab] = useState<string>("");
+
+  useEffect(() => {
+    setKeyActiveTab(pathUrl);
+  }, [pathUrl]);
+  // const { pid } = router.query;
+  // console.log("vwvw",router.asPath);
+  let page;
+  switch (pathUrl) {
+    case "/customer/account":
```
Comment (nguyenhoaidanh) [inline ecommerge_client/pages/customer/[pid].tsx:22]:
```
const APP_PATHS = {
   PROFILE:'/customer/account',
   ORDER_HISTORY:"/order/history",
   ...
}
```

File: ecommerge_client/components/TabMenu.tsx (hunk 3 above)
Comment (congtran8991) [reply ecommerge_client/components/TabMenu.tsx]: done

File: ecommerge_client/pages/customer/[pid].tsx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    case PROFILE:
+      page = <Account />;
+      break;
+    case ORDER_HISTORY:
+      page = <History />;
+      break;
+    case COUPONS:
+      page = <Coupons />;
+      break;
+    case "history":
```
Comment (nguyenhoaidanh) [inline ecommerge_client/pages/customer/[pid].tsx:32]: duplicate case

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**